js/apps/admin-ui/src/realm-settings/security-defences/BruteForceDetection.tsx

```diff
@@ -152,6 +156,50 @@ export const BruteForceDetection = ({
               bruteForceMode ===
                 BruteForceMode.PermanentAfterTemporaryLockout) && (
               <>
+                <FormGroup
+                  label={t("bruteForceStrategy")}
+                  fieldId="kc-brute-force-strategy"
+                  labelIcon={
+                    <HelpItem
+                      helpText={t("bruteForceStrategyHelp")}
+                      fieldLabelId="bruteForceStrategy"
+                    />
+                  }
+                >
+                  <Controller
+                    name="bruteForceStrategy"
+                    defaultValue="MULTIPLE"
+                    control={control}
+                    render={({ field }) => (
+                      <Select
+                        toggleId="kc-brute-force-strategy"
+                        onToggle={() =>
+                          setBruteForceStrategyOpen(!bruteForceStrategyOpen)
+                        }
+                        onSelect={(_, value) => {
+                          field.onChange(value as string);
+                          setBruteForceStrategyOpen(false);
+                        }}
+                        selections={field.value}
+                        variant={SelectVariant.single}
+                        data-testid="select-brute-force-strategy"
+                        aria-label={t("bruteForceStrategy")}
+                        isOpen={bruteForceStrategyOpen}
+                      >
+                        {bruteForceStrategyTypes.map((strategy) => (
+                          <SelectOption
+                            selected={strategy === field.value}
+                            key={strategy}
+                            value={strategy}
+                          >
+                            {t(`${strategy}`)}
+                          </SelectOption>
+                        ))}
+                      </Select>
+                    )}
+                  />
+                </FormGroup>
```

Comment:
This can be replaced with the `SelectControl` component from the ui-shared package:
```suggestion
                <SelectControl
                  name="bruteForceStrategy"
                  label={t("bruteForceStrategy")}
                  labelIcon={t("bruteForceStrategyHelp")}
                  controller={{ defaultValue="MULTIPLE" }}
                  options={bruteForceStrategyTypes} 
                />
```